pandas/types/common.py

```diff
@@ -73,7 +73,7 @@ def is_datetimetz(array):
 
 def is_period(array):
     """ return if we are a period array """
-    return isinstance(array, ABCPeriodIndex) or is_period_arraylike(array)
```

Comment:
hmm so this is now the same as ``is_period_dtype``, maybe we should just deprecate this then, unless this is *really* equiv to ``isinstance(val, Period)`` ? IOW a scalar